### DISCUSSION TITLE
Version bump to v4.0.11

### DISCUSSION THREAD

**Discussion by jasonhebert** [amazon-php/sp-api-sdk#251] [General]
Yo @norberttech when can we expect a bump to v4.0.11 to bring in the updates from the past month?

We'd really appreciate it and appreciate you!

Thanks...

BTW: I filed a case with Amazon in their SellerCentral support app for some of the issues we've seen here and had to make workarounds for. No response or activity whatsoever on the Github issue I submitted. :(

**Comment by norberttech** [comment]
https://github.com/amazon-php/sp-api-sdk/releases/tag/4.0.11 sorry @jasonhebert! 
We have been waiting with @owsiakl to verify the stability of the recent changes as they are mostly related to differences between the open API model specs and what amazon really returns. It seems we are good, at least for now. 
We also do talk to amazon directly about those issues, each discrepancy is reported but so far it does not look like they can resolve them.

**Reply by jasonhebert** [reply]
I have filed support tickets with Amazon in Seller Central as well. Their responses have been so far less than helpful and usually just point to the documentation. The Github issues I've filed have gone unanswered, probably unseen.

I guess the only thing we can do is keep up the good fight and chug on.

Thanks for the things you do and I'll help in any way that I can!
